src/scripts/components/widgets/PlayerPlugins/ProfileButton.js

```diff
@@ -0,0 +1,32 @@
+/* @flow */
+
+import React, { Fragment } from 'react'
+import ReactDOM from 'react-dom'
+
+import IconButton from 'components/foundations/buttons/IconButton'
+import Popover from 'components/foundations/Popover'
+
+type Props = {
+  popoverPortal: HTMLElement,
+  popoverOpen: boolean,
+  onClick: (e: Object) => void
+}
+
+const ProfileButton = (props: Props) => {
+  const { popoverPortal, popoverOpen, onClick } = props
+
+  return (
+    <Fragment>
+      <IconButton
+        data-test-id="overlay-profile-button"
```

Comment:
Given that `styled-components` class names are not something we can rely on in our tests, I propose we start using a `data-test-id` in our components for testing purposes when necessary. Thoughts?